ansible_tower_aws/1_provision.yml

```diff
@@ -8,6 +8,21 @@
   tasks:
   - name: explain how to login
     debug:
-      msg: "Please run 'ansible-playbook 2_preload.yml' to load the managed nodes."
+      msg: "Please run 'ansible-playbook 2_preload.yml' to load the AWS-hosted install server, or 'ansible-playbook 3_load.yml' to directly load the managed nodes."
```

Comment:
Does it make sense to move this task after the acme.sh section for signed certs? Just so that it prints as the last thing on the user's screen?